ext/odbc/odbc.stub.php

```diff
@@ -364,158 +364,85 @@ function odbc_fetch_array($statement, ?int $row = null): array|false {}
  */
 function odbc_fetch_into($statement, &$array, ?int $row = null): int|false {}
 
-/** @param resource $statement */
-function odbc_fetch_row($statement, ?int $row = null): bool {}
+    function odbc_fetch_row(ODBC\Result $statement, ?int $row = null): bool {}
 
-/** @param resource $statement */
-function odbc_result($statement, string|int $field): string|bool|null {}
+    function odbc_result(ODBC\Result $statement, string|int $field): string|bool|null {}
 
-/**
- * @param resource $statement
- * @deprecated
- */
-function odbc_result_all($statement, string $format = ""): int|false {}
+    /** @deprecated */
+    function odbc_result_all(ODBC\Result $statement, string $format = ""): int|false {}
 
-/** @param resource $statement */
-function odbc_free_result($statement): true {}
+    function odbc_free_result(ODBC\Result $statement): true {}
 
-/**
- * @return resource|false
- */
-function odbc_connect(string $dsn, ?string $user = null, #[\SensitiveParameter] ?string $password = null, int $cursor_option = SQL_CUR_USE_DRIVER) {}
+    function odbc_connect(string $dsn, ?string $user = null, #[\SensitiveParameter] string $password, int $cursor_option = SQL_CUR_USE_DRIVER): ODBC\Connection|false {}
 
-/**
- * @return resource|false
- */
-function odbc_pconnect(string $dsn, ?string $user = null, #[\SensitiveParameter] ?string $password = null, int $cursor_option = SQL_CUR_USE_DRIVER) {}
+    function odbc_pconnect(string $dsn, ?string $user = null, #[\SensitiveParameter] string $password, int $cursor_option = SQL_CUR_USE_DRIVER): ODBC\Connection|false {}
```

Comment:
I don't think there will be any such issues anymore, since these are the only two functions which had more nullable params since my original commit. I'll try to double check the stub just in case though.